python/hail/expr/aggregators.py

```diff
@@ -1087,3 +1088,64 @@ def info_score(gp) -> StructExpression:
     """
     t = hl.tstruct(score=hl.tfloat64, n_included=hl.tint32)
     return _agg_func('infoScore', gp, t)
+
+@typecheck(y=agg_expr(expr_float64),
+           xs=oneof(expr_float64, sequenceof(expr_float64)))
+def linreg(y, xs):
+    """Compute linear regression statistics.
+
+    Examples
+    --------
+
```

Comment:
why this extra line? we don't have it in the `Notes` section. we always disagree on this, let's settle it!